compiler/rustc_smir/src/rustc_internal/mod.rs

```diff
@@ -163,3 +166,38 @@ pub type Opaque = impl Debug + ToString + Clone;
 pub(crate) fn opaque<T: Debug>(value: &T) -> Opaque {
     format!("{value:?}")
 }
+
+pub struct StableMir {
+    args: Vec<String>,
+    test_function: fn(TyCtxt<'_>),
```

Comment:
I'm not sure what would be a good name for this one, but maybe `process` for now?

Reply:
Yeah, wasn't sure about the naming

Reply:
or callback